schema/schemaspec/spec.go

```diff
@@ -68,16 +68,17 @@ type (
 	// TypeSpec represents a specification for defining a Type.
 	TypeSpec struct {
 		// Name is the identifier for the type in an Atlas DDL document.
-		Name       string
-		 // T is the database identifier for the type.
+		Name string
+		// T is the database identifier for the type.
 		T          string
 		Attributes []*TypeAttr
```

Comment:
```suggestion
		Attrs []*TypeAttr
```

We use Attrs in all other places in atlas.